release_dashboard/lib/services/conductor.dart

```diff
@@ -40,4 +41,9 @@ abstract class ConductorService {
     required String incrementLetter,
     required String releaseChannel,
   });
+
+  /// Deletes the current release in progress by deleting the state file.
```

Comment:
Refrain from explaining the underlying implementation in the first sentence.

`Deletes the current release state.`

Since this logic is in conductor core, it may change (such as using RPC to clear a database)